apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -760,26 +718,24 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
 
     const resolveHelpPanel = () => {
 
-        if (showAsStandaloneIdentityProvider && currentWizardStep !== 0) return null;
-        if (!showAsStandaloneIdentityProvider && currentWizardStep !== 1) return null;
+        const SECOND_STEP: number = 1;
+        if (currentWizardStep !== SECOND_STEP) return null;
```

Comment:
It's better to move the constant to the top of the page. So that in case we need to access, it's available for the entire file.